.vscode/cspell.json

```diff
@@ -141,6 +141,7 @@
     "referer",
     "renormalize",
     "retriable",
+    "Rdma",
```

Comment:
probably add this only for your service, like L182-L186

Reply:
This is not updated though

Reply:
Will move to service specific section, as suggested.- 
{
      "filename": "**/sdk/computefleet/**/*.cs",
      "words": [
        "Rdma"
      ]
    },

Reply:
Updated as suggested.